src/components/ChatSection/ChatWindow/ChatItem/MessageItem/MessageItem.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import classes from "./MessageItem.module.sass";
+import moment from "moment";
+
+const MessageItem = ({ messageItem, currentUserId }) => {
```

Comment:
you can simplify it like that
const MessageItem = ({ messageItem, currentUserId }) => ({
  // put here jsx
});